src/lib/Resources/config/container/solr/criterion_visitors.yml

```diff
@@ -348,3 +348,7 @@ services:
         tags:
             - { name: ibexa.search.solr.query.content.criterion.visitor }
             - { name: ibexa.search.solr.query.location.criterion.visitor }
+
+    Ibexa\Solr\Query\Common\CriterionVisitor\IsUserEnabled:
+        tags:
+            - { name: ibexa.search.solr.query.content.criterion.visitor }
```

Comment:
Aren't we missing `- { name: ibexa.search.solr.query.location.criterion.visitor }`?

Reply:
I'd assume `Location` objects do not index the relevant data, so this criterion cannot be used with one.